src/ArchGDAL.jl

```diff
@@ -5,6 +5,7 @@ using GDAL: GDAL
 using GeoFormatTypes: GeoFormatTypes
 import GeoInterface
 using GeoInterfaceRecipes
+using GeoInterfaceMakie: GeoInterfaceMakie as GIM
```

Comment:
Can we continue to spell out `GeoInterfaceMakie` in full without abbreviating it as `GIM`?

Reply:
Yes I can do that. Another possibility would be to only rename `@enable` to `@enable_makie` because it is the only thing that we need from GeoInterfaceMakie.